packages/rum-core/src/domain/rumSessionManager.ts

```diff
@@ -58,11 +60,14 @@ export function startRumSessionManager(
       }
       return {
         id: session.id,
-        sessionReplayAllowed: session.trackingType === RumTrackingType.TRACKED_WITH_SESSION_REPLAY,
+        sampledForReplay: session.trackingType === RumTrackingType.TRACKED_WITH_SESSION_REPLAY,
+        sessionReplayAllowed:
+          session.trackingType === RumTrackingType.TRACKED_WITH_SESSION_REPLAY || session.isReplayForced,
```

Comment:
**💬 suggestion:** ‏Maybe a single property could be enough:
```suggestion
        sessionReplay:
          session.trackingType === RumTrackingType.TRACKED_WITH_SESSION_REPLAY ? 
            SessionReplayState.SAMPLED : 
          session.isReplayForced ?
            SessionReplayState.FORCED :
            SessionReplayState.OFF
```

Reply:
Nice suggestion. Done ✔️